Classes/Application/Command/NodeTemplateCommandController.php

```diff
@@ -42,4 +64,90 @@ public function createFromNodeSubtreeCommand(string $startingNodeId, string $wor
         }
         echo $this->nodeTemplateDumper->createNodeTemplateYamlDumpFromSubtree($node);
     }
+
+    /**
+     * Checks if all configured NodeTemplates are valid. E.g no syntax errors in EEL expressions,
+     * that properties exist on the node type and their types match and other checks.
+     *
+     * We process and build all configured NodeType templates. No nodes will be created in the Content Repository.
+     *
+     */
+    public function validateCommand(): void
+    {
+        $templatesChecked = 0;
+        /**
+         * nodeTypeNames as index
+         * @var array<string, array{caughtExceptions: CaughtExceptions, dataWasAccessed: bool}> $faultyNodeTypeTemplates
+         */
+        $faultyNodeTypeTemplates = [];
+
+        foreach ($this->nodeTypeManager->getNodeTypes(false) as $nodeType) {
+            $templateConfiguration = $nodeType->getOptions()['template'] ?? null;
+            if (!$templateConfiguration) {
+                continue;
+            }
+            $caughtExceptions = CaughtExceptions::create();
```

Comment:
I am unsure about the control flow here - I pass Caught Exceptions into here, so that they are filled? (why can't they be returned as the result of "processTemplateConfiguration"?

Reply:
As discussed, its the simpler choice here and more fun to work with for example inside `TemplatePart::processConfiguration`